display/display.go

```diff
@@ -291,6 +380,33 @@ func Init(version, commit, builtBy string) {
 		// keybinding in config/config.go and update the help panel in display/help.go
 
 		cmd := config.TranslateKeyEvent(event)
+		if tabs[curTab].mode == tabModeSearch {
+			switch cmd {
+			case config.CmdNextMatch:
+				if curMatch < (matches - 1) {
+					curMatch++
+					tabs[curTab].view.Highlight(fmt.Sprint("search-", curMatch))
+				}
+				tabs[curTab].view.ScrollToHighlight()
+				return nil
+			case config.CmdPrevMatch:
+				if curMatch > 0 {
+					curMatch--
+					tabs[curTab].view.Highlight(fmt.Sprint("search-", curMatch))
+				}
+				tabs[curTab].view.ScrollToHighlight()
+				return nil
+			case config.CmdInvalid:
+				if event.Key() == tcell.KeyEsc {
```

Comment:
You can ignore this case and just check the key with an if statement outside the switch.